src/utils/validator.ts

```diff
@@ -73,5 +73,27 @@ export const validators = [
         moment(field.value).isBefore(moment().add(field[UI.MAX_DATE_OFFSET], 'd')),
         field[UI.VALIDATION_ERROR]
       )
+  },
+  {
+    name: UI.MIN_DATE_TIME_OFFSET,
+    action: (field: any) => {
+      if (field[UI.MIN_VALUE]) return actionMethod(true, 'Skipped validation')
+
+      return actionMethod(
+        moment(field.value).isAfter(moment().subtract(field[UI.MIN_DATE_TIME_OFFSET], 'seconds')),
```

Comment:
According to the doc, the mindateOffset is in dates, not seconds, should this be updated or is it being transformed?

Reply:
You are correct that `mindateOffset` is in days. However this is `mindatetimeOffset` which is in seconds.